sbatch_scripts/many_small_jobs.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#SBATCH --account=windse
+#SBATCH --time=48:00:00
+#SBATCH --nodes=1
+#SBATCH --job-name=many_small_jobs
+
+ 
+# Load and activate your conda environment
+module purge
+module load conda
+source activate windse
+ 
+# This stops the solver from filling all 36 cores
+# Not using this may significantly slow down execution (100 times slower on Eagle)
+export OMP_NUM_THREADS=1
+
+
+# Number of times all processors run a job = # of jobs/ # of cores per node
+N_LOOP=2
+# Run 1 job per task
+N_JOB_PER_LOOP=$SLURM_NTASKS
+
+# Where to store Logs
+mkdir -p LOGS
+
+
+# Run many small jobs
+for((j=0;j<$N_LOOP;j++))
+do
+
+  for((i=0;i<$N_JOB_PER_LOOP;i++))
+  do
+
+    jobid=$((i + j*N_JOB_PER_LOOP))
+    windse run test.yaml -p 'general':'name':test_$jobid > LOGS/log$jobid &
```

Comment:
Likewise, although we don't need it here, it could be helpful to show how `srun` fits into this picture in the even the user wants to start 9 x 4-core jobs.
```bash
# Run each as a 1-core job, where -n XX can be adjusted as needed
# (e.g., srun -n 2... where N_JOB_PER_LOOP=18)
srun -n 1 windse run test.yaml...
```

Reply:
This is added in the newest PR. To be explicit, I replaced ``LOOP" with ``BATCH"